bin/_docker.sh

```diff
@@ -53,11 +59,18 @@ docker_build() {
       if [ -n "$DOCKER_BUILDKIT_CACHE" ]; then
         cache_params="--cache-from type=local,src=${DOCKER_BUILDKIT_CACHE} --cache-to type=local,dest=${DOCKER_BUILDKIT_CACHE},mode=max"
       fi
-      log_debug "  :; docker buildx $rootdir $cache_params --load -t $repo:$tag -f $file $*"
+
+      output_params="--load"
+      if [ -n "$DOCKER_MULTIARCH" ]; then
+        output_params="--push --platform $SUPPORTED_ARCHS"
+      fi
```

Comment:
This has implicit the need of pushing when building for more than one platform (as required per docker/buildx#59). Can you make this more explicit? Like using `--push` only if `DOCKER_PUSH` is set, and `--platform` if `DOCKER_MULTIARCH` is set. And before that, failing with an error if the latter is set while the former isn't (with a comment linking to the buildx issue explaining why we need to do this).